### ISSUE TITLE
Mixed Content on View/Edit Cluster Config

### ISSUE THREAD

**Issue by cronventis** [yesinteractive/kong-map#15]
When opening the View/Edit Cluster Config page if kongmap is hosted in k8s on port 80 and made available via kong with an ssl certificate, the page will not load since ace.js is loaded via http:

```
Mixed Content: The page at 'https://kongmap.domain.de/deck/edit/1/0' was loaded over HTTPS, but requested an insecure script 'http://ajaxorg.github.io/ace-builds/src-noconflict/ace.js'. This request has been blocked; the content must be served over HTTPS.
```

**Comment by cronventis** [comment]
Is this project still active?
Please make it open source so PR's are possible.. Would be a shame if the development stops....

**Comment by nikirago** [comment]
resolved in build 20210706
